src/NexusMods.App.UI/WorkspaceSystem/Page/PageFactoryController.cs

```diff
@@ -43,12 +49,15 @@ public Page Create(PageData pageData, WorkspaceId workspaceId, PanelId panelId,
         return page;
     }
 
-    public IEnumerable<PageDiscoveryDetails> GetAllDetails()
+    /// <summary>
+    /// Returns all <see cref="PageDiscoveryDetails"/> of every factory given a <see cref="IWorkspaceContext"/>
+    /// </summary>
+    public IEnumerable<PageDiscoveryDetails> GetAllDetails(IWorkspaceContext workspaceContext)
```

Comment:
This is still not very clear, what are these `PageDiscoveryDetails`, as in what are they for?
We don't have a good explanation of the system and how it is supposed to be used.

Should we add a Doc page explaining how it all works? Or would you prefer to try to make things clearer by commenting the code more with details on purpose and how they are meant to be used?